### PR TITLE
Figures chapter 5 issue #109

### PR DESCRIPTION
**Pull request by aurorarossi**

### CODE REVIEW

File: src/World2/world2/scenarios.jl [hunk 1]
```diff
@@ -128,3 +128,161 @@ function food_shortage(; kwargs...)
...
+function less_pollution_and_higher_agricolture_productivity(; kwargs...)
+    nr_parameters = NaturalResources.getparameters()
+    nr_parameters[:nrun1] = 0.25
+
+    pol_parameters = Pollution.getparameters()
+    pol_parameters[:poln1] = 0.7
+
+    agr_parameters = AgricultureInvestment.getparameters()
+    agr_parameters[:fc1] = 1.25
+
```
Comment (paulobruno) [inline src/World2/world2/scenarios.jl:213]: There are blank spaces in this line, probably remnants of auto indent.

File: src/World2/world2/scenarios.jl [hunk 2]
```diff
@@ -128,3 +128,161 @@ function food_shortage(; kwargs...)
...
+    ci_parameters = CapitalInvestment.getparameters()
+    ci_parameters[:cign1] = 0.06
+
```
Comment (paulobruno) [inline src/World2/world2/scenarios.jl:234]: Blank spaces in this empty line also.

File: src/World2/world2/scenarios.jl [hunk 3]
```diff
@@ -128,3 +128,161 @@ function food_shortage(; kwargs...)
...
+    pol_parameters = Pollution.getparameters()
+    pol_parameters[:poln1] = 0.4
+
+    agr_parameters = AgricultureInvestment.getparameters()
+    agr_parameters[:fc1] = 1.25
+
+    ci_parameters = CapitalInvestment.getparameters()
+    ci_parameters[:cign1] = 0.06
+
```
Comment (paulobruno) [inline src/World2/world2/scenarios.jl:256]: Blank spaces in empty line.

**Review by paulobruno — CHANGES_REQUESTED** [review]

**Comment by paulobruno** [comment]
There is also a problem with the size of function names. But this is a broader problem of lacking a well-defined styling guide. So, you can leave the big names and we change them later when we have a proper naming convention.

**Comment by aurorarossi** [comment]
> There is also a problem with the size of function names. But this is a broader problem of lacking a well-defined styling guide. So, you can leave the big names and we change them later when we have a proper naming convention.

Yes, maybe we should put something like scenario_number of the figure that use it.

**Pull request merged**